model/host/db.go

```diff
@@ -205,18 +205,35 @@ func IdleEphemeralGroupedByDistroID() ([]IdleHostsByDistroID, error) {
 func runningHostsQuery(distroID string) bson.M {
 	query := IsLive()
 	if distroID != "" {
-		key := bsonutil.GetDottedKeyName(DistroKey, distro.IdKey)
-		query[key] = distroID
+		query[bsonutil.GetDottedKeyName(DistroKey, distro.IdKey)] = distroID
 	}
 
 	return query
 }
 
+func provisioningHostsQuery(distroID string) bson.M {
```

Comment:
The naming is confusing since this doesn't actually query for provisioning hosts - could you call this something like startedHostsQuery? Same with the other functions